source/linear-algebra/source/01-LE/03.ptx

```diff
@@ -253,27 +265,51 @@ we can immediately tell how many solutions the system has.
           <li>
               <p>
 If the linear system given by <m>\RREF(A)</m> includes the contradiction
-<m>0=1</m>, that is, the row
-<m>\left[\begin{array}{ccc|c}0&amp;\cdots&amp;0&amp;1\end{array}\right]</m>,
+<me>0=1</me>, that is, the <m>\RREF</m> matrix includes the row
+<me>\left[\begin{array}{ccc|c}0&amp;\cdots&amp;0&amp;1\end{array}\right]</me>,
 then the system is <em>inconsistent</em>, which means it has <em>zero</em> solutions
-and its solution set is written as <m>\emptyset</m> or <m>\{\}</m>.
+and we may write
+<me>
+    \text{Solution set }=\{\}
+    \hspace{2em}\text{or}\hspace{2em}
+    \text{Solution set }=\emptyset.
+</me>
               </p>
           </li>
           <li>
               <p>
 If the linear system given by <m>\RREF(A)</m> sets each variable of the system
-to a single value; that is, <m>x_1=s_1</m>, <m>x_2=s_2</m>, and so on through
-<m>x_n=s_n</m>;
-then the system is <em>consistent</em> with exactly <em>one</em> solution
-<m>\left[\begin{array}{c}s_1\\s_2\\\vdots\\s_n\end{array}\right]</m>,
-and its solution set is
-<m>\setList{ \left[\begin{array}{c}s_1\\s_2\\\vdots\\s_n\end{array}\right] }</m>.
+to a single value; that is we have:
+
+<md alignat-columns='2' alignment="alignat">
+    <mrow>
+    x_1 &amp;= s_1
+    </mrow>
+    <mrow>
+    x_2 &amp;= s_2
+    </mrow>
+    <mrow>
+    &amp;\vdots
+    </mrow>
+    <mrow>
+    x_n &amp;= s_n
+    </mrow>
+</md>
+(with some possible extra <m>0=0</m> equations),
+then the system is <em>consistent</em> with exactly <em>one</em> solution,
+and we may write
+<me>
+    \text{Solution }=\left[\begin{array}{c}s_1\\s_2\\\vdots\\s_n\end{array}\right]
+    \hspace{2em}\text{but}\hspace{2em}
+    \text{Solution set }=
+        \left\{\left[\begin{array}{c}s_1\\s_2\\\vdots\\s_n\end{array}\right]\right\}.
+</me>
               </p>
           </li>
           <li>
               <p>
-Otherwise, the system must have more variables than non-trivial
-equations (equations other than <m>0=0</m>). This means it is
+Otherwise, the system given by the <m>\RREF</m> matrix must <em>not include a <m>0=1</m> contradiction</em>
+while <em>including at least one equation with multiple variables</em>. This means it is
 <em>consistent</em> with <em>infinitely-many</em>
 different solutions. We'll learn how to find such solution sets in 
 <xref ref="LE4"/>. 
```

Comment:
I think we overcomplicated it previously: we have $\infty$-many solutions as long as the RREF has no contradictions, and at least one equation with multiple variables. No need to compare count of non-trivial equations vs variables: I just rule out 0 or 1 solution, then know it has $\infty$-many.